ci/docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+version: "3.9"
+
+services:
+  node1:
+    image: ghcr.io/xline-kv/xline:latest
+    networks:
+      xline_network:
+        ipv4_address: 172.22.0.2
+    volumes:
+      - ./fixtures:/mnt
+    ports:
+      - "2479:2379"
+    environment:
+      RUST_LOG: curp=debug,xline=debug
+    command: >
+      xline
+      --name node1
+      --members node1=http://172.22.0.2:2379,node2=http://172.22.0.3:2379,node3=http://172.22.0.4:2379
+      --storage-engine rocksdb
+      --data-dir /usr/local/xline/data-dir
+      --auth-public-key /mnt/public.pem
+      --auth-private-key /mnt/private.pem
+      --client-listen-urls http://172.22.0.2:2379
+      --peer-listen-urls http://172.22.0.2:2380
+
+  node2:
+    image: ghcr.io/xline-kv/xline:latest
+    networks:
+      xline_network:
+        ipv4_address: 172.22.0.3
+    volumes:
+      - ./fixtures:/mnt
+    ports:
+      - "2380:2379"
+    environment:
+      RUST_LOG: curp=debug,xline=debug
+    command: >
+      xline
+      --name node2
+      --members node1=http://172.22.0.2:2379,node2=http://172.22.0.3:2379,node3=http://172.22.0.4:2379
+      --storage-engine rocksdb
+      --data-dir /usr/local/xline/data-dir
+      --auth-public-key /mnt/public.pem
+      --auth-private-key /mnt/private.pem
+      --client-listen-urls http://172.22.0.3:2379
+      --peer-listen-urls http://172.22.0.3:2380
+
+  node3:
+    image: ghcr.io/xline-kv/xline:latest
+    networks:
+      xline_network:
+        ipv4_address: 172.22.0.4
+    volumes:
+      - ./fixtures:/mnt
+    ports:
+      - "2579:2379"
+    environment:
+      RUST_LOG: curp=debug,xline=debug
+    command: >
+      xline
+      --name node3
+      --members node1=http://172.22.0.2:2379,node2=http://172.22.0.3:2379,node3=http://172.22.0.4:2379
+      --storage-engine rocksdb
+      --data-dir /usr/local/xline/data-dir
+      --auth-public-key /mnt/public.pem
+      --auth-private-key /mnt/private.pem
+      --client-listen-urls http://172.22.0.4:2379
+      --peer-listen-urls http://172.22.0.4:2380
+
+  client:
+    image: appropriate/curl # Using a lightweight client for demonstration purposes
```

Comment:
It should be ghcr.io/xline-kv/etcdctl:v3.5.9. This client container is used in the validation CI process. Please make the modification.

Reply:
ok.. i will rebase this pr and make the modification.